Calculators/Employee Salary Calculator/README.md

```diff
@@ -0,0 +1,27 @@
+## Description 📜
+The basic idea of this calculator is to provide the Net Salary by providing Basic Salary and Tax using HTML , CSS , Javascript.
+
+
+<hr>
+ 
+## Checklist ✅
+
+<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->
+
+- [x] I follow [Contributing Guidelines](https://github.com/vasu-1/CalcHub/blob/main/.github/ContributingGuidelines.md) & [Code of conduct](https://github.com/vasu-1/CalcHub/blob/main/.github/CODE_OF_CONDUCT.MD) of this project.
+- [x] I have performed a self-review of my own code or work.
+- [x] I have commented my code, particularly in hard-to-understand areas.
+- [x] My changes generates no new warnings.
+- [x] I'm GSSOC'22 contributor
+
+
+<hr>
+
+<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->
+
+## Screenshots / Gif (Optional) 📸
+![Screenshot (534)](https://user-images.githubusercontent.com/93437720/168834778-4e51cb43-9ddd-477b-9a4b-a144b63a9123.png)
```

Comment:
don't include these whole thing !
you just need to include the basic functionalities and what your calculator does !

Reply:
ok sure I will include